numpy/core/_add_newdocs.py

```diff
@@ -504,6 +504,8 @@
 
 add_newdoc('numpy.core', 'nested_iters',
     """
+    nested_iters(op, axes, flags=None, op_flags=None, op_dtypes=None, order="K", casting="safe", buffersize=0)
```

Comment:
I don't think there is any way here around the "line too long" complaints, is there?

Reply:
Hmm. Don't know of one for sure, but it looks like line continuation works inside the quotes
```
>>> """
... Hello \
... World
... """
'\nHello World\n'
```
The problem might be the indentation of the next line cannot be aligned with the opening "(", but that might not matter.

Reply:
Good idea, it seems adding the line-break did the trick:
https://22298-908607-gh.circle-artifacts.com/0/doc/build/html/reference/generated/numpy.nested_iters.html?highlight=nested_iters#numpy.nested_iters